tools/vms-generator/utils/utils.go

```diff
@@ -99,6 +100,12 @@ const defaultInterfaceName = "default"
 const enableNetworkInterfaceMultiqueueForTemplate = true
 const EthernetAdaptorModelToEnableMultiqueue = "virtio"
 
+const (
+	cloudConfigHeader                 = "#cloud-config"
+	cloudConfigInstallAndStartService = "packages:\n  - nginx\nruncmd:\n  - [ \"systemctl\", \"enable\", \"--now\", \"nginx\" ]"
+	cloudConfigUserPassword           = "password: fedora\nchpasswd: { expire: False }"
+)
+
```

Comment:
I would convert those strings to be multi-line instead of adding \n

Reply:
Done. Fair point.